akka-docs/src/main/paradox/project/downstream-upgrade-strategy.md

```diff
@@ -0,0 +1,42 @@
+---
+project.description: Upgrade strategy for downstream libraries
+---
+# Downstream upgrade strategy
+
+When a new Akka version is released, downstream projects (such as
+[Akka Management](https://doc.akka.io/docs/akka-management/current/),
+[Akka HTTP](https://doc.akka.io/docs/akka-http/current/) and
+[Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/))
+do not need to update immediately: because of our
+@ref[binary compatibility](../common/binary-compatibility-rules.md) approach,
+applications can take advantage of the latest version of Akka without having to
+wait for intermediate libraries to update.
+
+## Patch versions
+
+When releasing a new patch version of Akka (e.g. 2.5.22), we typically don't
+immediately bump the Akka version in sattelite projects.
+
+The reason for this is this will make it more low-friction for users to update
+those sattelite projects: say their project is on akka 2.5.22 and
+akka-management 1.0.0, and we release akka-management 1.0.1 (still against
+Akka 2.5.22) and Akka 2.5.23. They can safely update to akka-managemenet 1.0.1
+without also updating to akka 2.5.23.
```

Comment:
Akka

Reply:
Shall we also clarify the other scenario where they can update to Akka 2.5.23 but still use akka-management 1.0.0 (or 1.0.1)?